lib/eppo.dart

```diff
@@ -0,0 +1,77 @@
+export 'src/configuration_wire_protocol.dart' show FormatEnum, VariationType;
+export 'src/subject.dart' show Subject, ContextAttributes;
+export 'src/constants.dart' show defaultLoggerPrefix;
+export 'src/precompute_client.dart'
+    show
+        EppoPrecomputedClient,
+        SdkOptions,
+        PrecomputeArguments,
+        FlagEvaluation,
+        Variation,
+        BanditEvaluation;
+export 'src/api_client.dart' show EppoApiClient;
+export 'src/http_client.dart' show EppoHttpClient;
+export 'src/sdk_version.dart' show SdkPlatform;
+export 'src/configuration_store.dart' show ConfigurationStore;
+export 'src/crypto.dart' show getMD5Hash;
+export 'src/assignment_logger.dart' show AssignmentLogger, AssignmentEvent;
+export 'src/assignment_cache.dart'
+    show
+        AssignmentCache,
+        AssignmentCacheKey,
+        CacheValue,
+        VariationCacheValue,
+        BanditCacheValue,
+        AssignmentCacheEntry,
+        InMemoryAssignmentCache,
+        NoOpAssignmentCache;
+export 'src/bandit_logger.dart' show BanditLogger, BanditEvent;
+
+import 'src/precompute_client.dart';
+
+/// The main interface to interact with the Eppo SDK
```

Comment:
This file is the equilivant to `index.js` and the main entrypoint for exporting types